bundles/org.openhab.binding.androiddebugbridge/src/main/java/org/openhab/binding/androiddebugbridge/internal/AndroidDebugBridgeConfiguration.java

```diff
@@ -42,10 +42,18 @@ public class AndroidDebugBridgeConfiguration {
      * Record input duration in seconds.
      */
     public int recordDuration = 5;
+    /**
+     * Percent to increase/decrease volume.
+     */
+    public int volumeStepPercent = 15;
     /**
      * Assumed max volume for devices with android versions that do not expose this value (>=android 11).
      */
     public int deviceMaxVolume = 25;
+    /**
+     * Max ADB command consecutive timeouts to force to reset the connection.
+     */
+    public int maxADBTimeouts = 10;
```

Comment:
Why setting a default to 10 while the default in the XML is 0 ?

Reply:
My mistake, I used 10 initially but then realized it was better to add it disabled by default. Thank you!